src/SingleProject/Resizetizer/src/ResizetizeSharedImages.cs

```diff
@@ -232,6 +232,8 @@ List<SharedImageInfo> ParseImageTaskItems(ITaskItem[] images)
 				var info = new SharedImageInfo();
 
 				var fileInfo = new FileInfo(image.GetMetadata("FullPath"));
+				if (!fileInfo.Exists)
+					throw new FileNotFoundException("Unable to find background file: " + fileInfo.FullName, fileInfo.FullName);
```

Comment:
Here and below... Just throw. No need for clever things later on. The path was probably wrong.